eta/core/annotations.py

```diff
@@ -867,26 +1296,27 @@ def _draw_object(img, obj, annotation_config, color=None, pre_attrs=None):
     show_box = annotation_config.show_object_boxes
     show_name = (
         annotation_config.show_object_names or annotation_config.show_all_names
-    )
+    ) and not annotation_config.hide_all_names
     show_label = annotation_config.show_object_labels
     show_attrs = annotation_config.show_object_attrs
     show_confidence = (
         annotation_config.show_object_confidences
         or annotation_config.show_all_confidences
-    )
+    ) and not annotation_config.hide_all_confidences
     show_attr_names = (
         annotation_config.show_object_attr_names
         or annotation_config.show_all_names
-    )
+    ) and not annotation_config.hide_all_names
     show_attr_confidences = (
         annotation_config.show_object_attr_confidences
         or annotation_config.show_all_confidences
-    )
+    ) and not annotation_config.hide_all_confidences
     show_index = annotation_config.show_object_indices
+    show_name_only_titles = annotation_config.show_name_only_titles
     occluded_attr = annotation_config.occluded_object_attr
     hide_occluded = annotation_config.hide_occluded_objects
-    labels_whitelist = annotation_config.object_labels_whitelist
-    labels_blacklist = annotation_config.object_labels_blacklist
+    labels_whitelist = annotation_config.labels_whitelist
+    labels_blacklist = annotation_config.labels_blacklist
```

Comment:
A versioning question: Since the field name change breaks compatibility (as far as I can tell), this would technically require a bump to `0.2.0`. Is that ok with you, or would you rather keep compatibility?
If this change doesn't break compatibility with FiftyOne/other library users, or if it's a small-enough change that it would go unnoticed, I would be fine sticking with `0.1.x`.

Reply:
I'm not too worried about backwards compatibility on this change, since it's unlikely that anyone is using this particular. Also, it wouldn't actually cause an error, only a silent ignore of a parameter:

```py
from eta.core.annotations import AnnotationConfig  # or fiftyone.utils.annotations

AnnotationConfig({"foo": "bar"})  # non-existent parameter gets silently ignored
```

Reply:
So I say stick with `0.1.X` here

Reply:
Good to know, that makes sense